src/export/Export.cpp

```diff
@@ -300,91 +289,6 @@ void Exporter::FixFilename()
-            int exportedChannels = -1;//undefined
-            if (exportFormat != wxT("CL") && exportFormat != wxT("FFMPEG"))
-               exportedChannels = mChannels;
-            
-            auto pWindow = ProjectWindow::Find(mProject);
-            if (exportedChannels == 1) {
-               if (ShowWarningDialog(pWindow,
-                  wxT("MixMono"),
-                  XO("Your tracks will be mixed down and exported as one mono file."),
-                  true) == wxID_CANCEL)
-                  return false;
-            }
-            else if (exportedChannels == 2) {
-               if (ShowWarningDialog(pWindow,
-                  wxT("MixStereo"),
-                  XO("Your tracks will be mixed down and exported as one stereo file."),
-                  true) == wxID_CANCEL)
-                  return false;
-            }
-            else {
-               if (ShowWarningDialog(pWindow,
-                  wxT("MixUnknownChannels"),
-                  XO("Your tracks will be mixed down to one exported file according to the encoder settings."),
-                  true) == wxID_CANCEL)
-                  return false;
-            }
-         }
-      }
-   }
-   else
-   {
-      constexpr auto MaxExportChannels = 32u;
-
-      auto exportTracks = ExportUtils::FindExportWaveTracks(TrackList::Get( *mProject ), mSelectedOnly);
-      mMixerSpec = std::make_unique<MixerOptions::Downmix>(exportTracks.size(),
-                                                           // JKC: This is an attempt to fix a 'watching brief' issue, where the slider is
-                                                           // sometimes not slidable.  My suspicion is that a mixer may incorrectly
-                                                           // state the number of channels - so we assume there are always at least two.
-                                                           // The downside is that if someone is exporting to a mono device, the dialog
-                                                           // will allow them to output to two channels. Hmm.  We may need to revisit this.
-                                                           // STF (April 2016): AMR (narrowband) and MP3 may export 1 channel.
-                                                           std::clamp(mPlugins[mFormat]->GetFormatInfo(mSubFormat).mMaxChannels,
-                                                                      1u,
-                                                                      MaxExportChannels));
-      if(prompt)
-      {
-         ExportMixerDialog md(exportTracks, mMixerSpec.get(),
-                              NULL,
-                              1,
-                              XO("Advanced Mixing Options"));
-         if(md.ShowModal() != wxID_OK)
-            return false;
-      }
-      mChannels = mMixerSpec->GetNumChannels();
```

Comment:
At the commit "Move `ExportMixerDialog` invocation and mix-related warnings", this line is deleted, and I don't see where it moves.